gtsfm/common/gtsfm_data.py

```diff
@@ -145,6 +145,23 @@ def add_camera(self, index: int, camera: PinholeCameraCal3Bundler) -> None:
             raise ValueError("Camera cannot be None, should be a valid camera")
         self._cameras[index] = camera
 
+    def get_track_length_statistics(self) -> Tuple[float, float, np.ndarray]:
+        """Compute mean and median lengths of all the tracks.
+
+        Returns:
+            Mean track length.
+            Median track length.
+            Array containing all track lengths.
```

Comment:
I think we should not return the full array with this function. There should be two functions.

Reply:
i think we often want all 3 things at once, though

Reply:
maybe rename it?